x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/slm/SnapshotLifecyclePolicyMetadata.java

```diff
@@ -51,6 +53,8 @@ public class SnapshotLifecyclePolicyMetadata implements SimpleDiffable<SnapshotL
     private final SnapshotInvocationRecord lastSuccess;
     @Nullable
     private final SnapshotInvocationRecord lastFailure;
+    @Nullable
+    private final Long invocationsSinceLastSuccess;
```

Comment:
How do you feel about making this non-nullable and instead treating a missing value as 0? I think that would be nicer code-wise since consumers don't have to handle a null value (and 0 invocations would be the "default" empty value)